### DISCUSSION TITLE
Coffee client

### DISCUSSION THREAD

**Discussion by SuperKayMan364** [community/community#56277] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This has been the best client I’ve ever used! Unlike any other clients, this has nbt features that I would really like to see being expanded on! Seeing you give up on the product is very sad because it was by far one of the most incredible clients I’ve ever seen or heard of. I hope you come back to this and continue because it’s very unique in its ability to tinkle with Minecraft’s vulnerabilities.

just in case you want to dm me, my discord is: “SKM364#0727“ and I would be glad to receive updates on the situation 

                Sincerely,
                SKM364

**Comment by rjr864073-dev** [comment]
It's always disappointing to see a project with so much potential stop receiving updates. Hopefully the developers revisit it in the future, because it clearly made a positive impression on a lot of users. Wishing the team all the best if they decide to continue development.
